### ISSUE TITLE
Disabled styles for Buttons and Links

### ISSUE THREAD

**Issue by peterkle** [xogroup/union#225]
We already have default colors for buttons and links, should we have styles for disabled ones? Like so:

![untitled](https://user-images.githubusercontent.com/486241/28681110-5fd5d398-72c6-11e7-9d50-e1e711ab9c5a.png)

I've set the color to gray and the pointer style to `not-allowed`

**Comment by jmsardina** [comment]
Yes!  We really need to have a different style for disabled states to quickly communicate the state of the button.  What's the status on the designs for this one?

**Comment by mlonsdale** [comment]
This is the prototype that Silvia had worked on for the CTAs (disabled styling shown at the bottom) https://xogroup.github.io/tk-form-fields/

Not sure if there's a reason that the disabled state wasn't originally included.

**Comment by GeorgeTaveras1231** [comment]
@mlonsdale there isn't a particular reason, at least from the engineering end. There were a lot of changes happening during that time that led to me accidentally missing this. 😅
